docs/docs/styling.md

```diff
@@ -61,3 +64,17 @@ As of sku v12, [treat] support has been removed in favour of [Vanilla Extract].
 
 [treat]: https://seek-oss.github.io/treat/
 [vanilla extract]: #vanilla-extract
+
+## External CSS
+
+CSS from third-party dependencies can be loaded using a side-effect import, e.g.
+
+```tsx
+import { SomeComponent } from 'some-package';
+
+import 'some-package/dist/styles.css';
+
+export const MyComponent = () => {
+  return <SomeComponent>{/* ... */}</SomeComponent>;
+};
+```
```

Comment:
Thoughts on adding a note about the scope?

````suggestion
```

Note, this should only be used when importing CSS dependencies from a third-party package in node_modules.
If you are writing custom CSS, please see the [Vanilla Extract](#vanilla-extract) section above.
````

Reply:
SGTM